assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -169,7 +169,11 @@ function createResourceAccessChart ({ data, width, height, domElement }) {
 
   function brushmove () {
     const fullRange = mainYZoom.range()
-    const selection = d3.event ? d3.event.selection : defaultSelection
+    const selection = d3.event
+      ? d3.event.selection[1] === 0 // prevents [0, 0] from being returned, which causes bug
```

Comment:
The bug was caused when the selection window's domain got to `[0, 0]`. I just check to make sure that the second element of the array is a 0, and if so set it to be small (0.1).